### ISSUE TITLE
Request for Naming/AsciiConstants

### ISSUE THREAD

**Issue by pjskennedy** [rubocop-hq/rubocop#7458]
Rubocop currently has [Naming/AsciiIdentifiers](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AsciiIdentifiers) which correctly adds offenses to ruby identifiers (`tIDENTIFIER`) containing non-ascii characters. I find that the same rigor should be applied to constants (`tCONSTANT`) as the following are currently not offenses:
```rb
class Foö
end
```
```rb
module Foö
end
```
```rb
FOÖ = "foo"
```

#### Possible Solution:
A new cop: `Naming/AsciiConstants` which adds offenses to code where non-ascii characters are used in constant declarations (See examples above)

#### Alternative: 
A configuration option for `Naming/AsciiIdentifiers` to incorporate constants. Offenses would exist for the examples above. However it does not appear that ruby considers constants as identifiers so this option could be misleading.
